packages/website/components/messagebanner/messagebanner.js

```diff
@@ -24,16 +24,19 @@ export default function MessageBanner() {
 
   let maintenanceMessage = '';
 
+  const incidents = statusPageData?.incidents || [];
+
   const scheduledMaintenances =
     statusPageData?.scheduled_maintenances.filter(
       (/** @type {{ status: string; }} */ maintenance) => maintenance.status !== 'completed'
     ) || [];
```

Comment:
Default to empty array _before_ using `filter`.

```suggestion
    (statusPageData?.scheduled_maintenances || []).filter(
      (/** @type {{ status: string; }} */ maintenance) => maintenance.status !== 'completed'
    );
```